examples/get.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require  dirname(__FILE__) . '/../src/sajari.php';
+
+use Sajari\WithEndpoint;
+use Sajari\Client;
+use Sajari\Key;
+
+$opts = [new WithEndpoint('server_address')];
+
+$c = new Client('myproject', 'mycollection', $opts);
+
+$key = new Key("_id", "7f7a8658-a368-d6f9-5e19-791debb6bddb");
+
+try {
+    $doc = $c->Get($key);
+} catch (Exception $e) {
```

Comment:
@tbillington people will want to be able to handle engine exceptions differently to the global `Exception`, so if possible, make sure that there's a nice way for this. Some options:

1. Make the code throw `EngineException`s so they can be caught separately.
2. Add some static helper methods like `Sajari::isEngineError($e)`, `Sajari::isKeyConflictError($e)` per the "don’t assert errors for type, assert for behaviour" recommendation in http://dave.cheney.net/2014/12/24/inspecting-errors